qiskit/pulse/builder.py

```diff
@@ -2210,9 +2253,9 @@ def call(
 
         .. warning::
 
-            Calling a circuit from a schedule is not encouraged. Currently, the Qiskit execution model
-            is migrating toward the pulse gate model, where schedules are attached to
-            circuits through the :meth:`.QuantumCircuit.add_calibration` method.
+            Calling a circuit from a schedule is deprecated as of qiskit 0.46.0. The Qiskit execution
+            model has migrating toward the pulse gate model, where schedules are attached to circuits
+            through the :meth:`.QuantumCircuit.add_calibration` method.
```

Comment:
```suggestion
            Calling a circuit from a schedule is deprecated as of qiskit 0.46.0. The Qiskit execution
            model is migrating towards the pulse gate model, where schedules are attached to circuits
            through the :meth:`.QuantumCircuit.add_calibration` method. Thus, instead of calling a circuit
            into the schedule, it should instead be added to the main circuit.
```

Reply:
Applied suggestion :)